cmd/ztest/ztest.c

```diff
@@ -866,7 +873,7 @@ process_options(int argc, char **argv)
 				usage(B_FALSE);
 			break;
 		case 'G':
-			ztest_dump_debug_buffer = B_TRUE;
+			zo->zo_dump_dbgmsg = 1;
```

Comment:
why did you change this from a boolean_t to an int?

Reply:
This was a minor thing. The variable in the struct is an int and I wanted to match `zo->zo_mmp_test`

Reply:
why did you change the variable from a boolean_t to an int?

Reply:
The variable needed to be moved into `zo` so that it could properly be shared between processes. This struct did not have any boolean fields, even for fields such as `zo_mmp_test` which are clearly logically boolean. I assumed this might have something to do with ensuring alignment so I was just following the precedent.